changelog/unreleased/changes/2119--log-actors-with-id.md

```diff
@@ -0,0 +1,2 @@
+Actor names in log messages now have an `-ID` suffix to make it easier to tell
+multiple instances of the same actor apart.
```

Comment:
Should this explain that the ID is globally unique, i.e., that index-22 does not mean that at least 22 index actors have existed?

Reply:
I see no reason to assume that from the given text.